library/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     buildToolsVersion "22.0.1"
 
     defaultConfig {
-        minSdkVersion 10
+        minSdkVersion 15
```

Comment:
Which SDK requires 15? I'd prefer to stay at the minimum the Firebase SDK itself requires if possible. If we need > 10, we should document (in a comment here) which functionality requires the newer version.